tests/testsuite/cache_messages.rs

```diff
@@ -441,18 +475,29 @@ fn rustc_workspace_wrapper() {
 
     p.cargo("check -v")
         .env("RUSTC_WORKSPACE_WRAPPER", tools::echo_wrapper())
-        .with_stderr_contains(
-            "WRAPPER CALLED: rustc --crate-name foo --edition=2015 src/lib.rs [..]",
+        .with_stderr_data(
+            "\
+[CHECKING] foo v0.0.1 ([ROOT]/foo)
+[RUNNING] [..]
```

Comment:
Should we instead have `[RUNNING] [..]/rustc-echo-wrapper rustc --crate-name foo [..]`, so that we are asserting it really runs rustc wrapper?

(And ditto in this test case)